### PR TITLE
[License] change license to Dual-License with GPL and Data Definition Commercial License (DDCL)

### PR DESCRIPTION
**Pull request by dpfaffenbauer**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -13,3 +22,13 @@ GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+## Data Definitions Commercial License (DDCL)
+Alternatively, commercial and supported versions of the program - also known as
+Commercial Distributions - must be used in accordance with the terms and conditions
+contained in a separate written agreement between you and CoreShop GmbH.
```
Comment (aarongerig) [inline LICENSE.md]: CoreShop GmbH should be replaced.

File: ecs.php [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+    $ecsConfig->ruleWithConfiguration(VisibilityRequiredFixer::class, ['elements' => ['const', 'property', 'method']]);
+
+    $header = <<<EOT
+This source file is available under two different licenses:
+ - GNU General Public License version 3 (GPLv3)
+ - Data Definitions Commercial License (DDCL)
+Full copyright and license information is available in
+LICENSE.md which is distributed with this source code.
+
+@copyright  Copyright (c) CORS GmbH (https://www.cors.gmbh) in combination with instride AG (https://www.instride.ch)
```
Comment (aarongerig) [inline ecs.php]: https://instride.ch without www

**Review by aarongerig — CHANGES_REQUESTED** [review]

File: LICENSE.md (hunk 1 above)
Comment (dpfaffenbauer) [reply LICENSE.md]: thanks done.

File: ecs.php (hunk 2 above)
Comment (dpfaffenbauer) [reply ecs.php]: thanks done.

**Comment by dpfaffenbauer** [comment]
@aarongerig please approve once you are fine with it

File: src/DataDefinitionsBundle/DependencyInjection/Compiler/CleanerRegistryCompilerPass.php [hunk 3]
```diff
@@ -22,12 +21,12 @@ final class CleanerRegistryCompilerPass extends RegisterSimpleRegistryTypePass
 {
     public const CLEANER_TAG = 'data_definitions.cleaner';
 
-    public function __construct()
-    {
+    public function __construct(
+        ) {
```
Comment (aarongerig) [inline src/DataDefinitionsBundle/DependencyInjection/Compiler/CleanerRegistryCompilerPass.php:25]: @dpfaffenbauer This seems to be a weird CS rule. Empty constructor should not be split across two lines and with that indentation IMO. WDYT?

Comment (dpfaffenbauer) [reply src/DataDefinitionsBundle/DependencyInjection/Compiler/CleanerRegistryCompilerPass.php:25]: empty constructor per-se should not exist. 

We use the same CS rule in CoreShop. I actually only added it for the License headers :D

Comment (aarongerig) [reply src/DataDefinitionsBundle/DependencyInjection/Compiler/CleanerRegistryCompilerPass.php:25]: Alright, then that's fine with me. :)

**Review by aarongerig — APPROVED** [review]

**Pull request merged**